api/src/routes/protected/classroom.ts

```diff
@@ -0,0 +1,147 @@
+import { FastifyPluginCallback } from 'fastify';
+import { Type } from '@fastify/type-provider-typebox';
```

Comment:
```suggestion
import { FastifyPluginCallbackTypebox } from '@fastify/type-provider-typebox';
```
We need this type because it uses the `schema` to generate types for the request and reply objects.